fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -669,6 +669,9 @@ public boolean canTxnFinished(TransactionState txn, Set<Long> errReplicas) {
                                 if (!errReplicas.contains(replica.getId()) && replica.getLastFailedVersion() < 0) {
                                     if (replica.getVersion() >= partitionCommitInfo.getVersion()) {
```

Comment:
The version of replica is updated through finish transaction or be report, and the result of canTxnFinished determines whether to execute finish transaction.
Load job maybe not able to finish normally when one backend is crash and quorum is satisfied.

Reply:
yes, Now the mechanism of visibility depend on the report of replica. It can be optimized.